### DISCUSSION TITLE
common bash use of fzf-tmux with an editor? (one that gets the right $LINES calculation?)

### DISCUSSION THREAD

**Discussion by jzacsh** [junegunn/fzf#2876] [Q&A]
**tl;dr** how to use fzf-tmux but give terminal a chance to recalculate dimensions? (`$LINES` is stale by the time my `$EDITOR` runs).

## exact steps

**background**: I have the following pattern of usage when calling `fzf-tmux`:

```sh
# in my bashrc:
$ function fzft() { fzf-tmux --preview='head --lines="$LINES" {}'; }
$ $EDITOR "$(fzft)"
```

**problem**:  what happens is that `$EDITOR` runs it doesn't get the right `$LINES` value (eg: it runs too fast and gets 59 instead of 119); I can see this is the issue if I tweak my example:

```sh
# in my bashrc:
$ function fzft() { fzf-tmux --preview='head --lines="$LINES" {}'; }
$ echo $LINES
119
$ fzft
.vim/ftplugin/ledger.vim # output from fzft
$ echo $LINES
119
$ fzft; echo $LINES # this emulates what happens in the real use-case described above; $LINES hasn't been recalculated yet...
.vim/ftplugin/ledger.vim # output from fzft
59

```

**Question**: has anyone else even seen this issue before? Or am I a special snowflake trying to open the result of an `fzf-tmux` print? :stuck_out_tongue: 

---

## pasting relevant info from the bug-report template:

- [x] I have read through the manual page (`man fzf`)
- [x] I have the latest version of fzf
- [x] I have searched through the existing issues

### Info

- a bit more about my environment:
    - a) `fzf --version` outputs `0.30`
    - b) `tmux -V` outputs `3.2a`
    - c) `nvim --version` outputs `0.7.2`
- OS
    - [x] Linux, specificaly debian testing
    - [ ] Mac OS X
    - [ ] Windows
    - [ ] Etc.
- Shell
    - [x] bash, specifically `5.1.8`
    - [ ] zsh
    - [ ] fish
